src/library.js

```diff
@@ -54,8 +54,12 @@ LibraryManager.library = {
   // Trivial wrappers around runtime functions that make these symbols available
   // to native code.
   segfault: function() { segfault(); },
-  alignfault: function() { alignfault(); },
   ftfault: function() { ftfault(); },
+  alignfault: function() {
+#if SAFE_HEAP == 1
+    alignfault();
+#endif
```

Comment:
Seems like this change is need need as the implementation of alignfault is already null.

Maybe revert this to keep this block of three functions simple and consistent?   Then can be considered simple forwarding function from libary.js to builtins, and not concerned with the implementation.

Reply:
+1